test/basic-images.test

```diff
@@ -151,13 +151,15 @@ check_ext() {
 	[ "$verbose" = "t" ] && set -x
 	# UUID is randomly generated
 	uuid="Filesystem UUID"
+	# depends on system architecture
+	flags="Filesystem flags"
 	# Hash Seed is randomly generated
 	seed="Directory Hash Seed:"
 	# checksum depends on random data
 	csum1="Checksum: \|Group 0: (Blocks 1-4095) csum"
 	# format change
 	csum2="Group 0: (Blocks 1-4095) \\[ITABLE_ZEROED\\]\|  Checksum .*, unused inodes 205"
-	dumpe2fs "${1}" | grep -v "^\($uuid\|$seed\|$csum1\|$csum2\)" > "dump" &&
+	dumpe2fs "${1}" | grep -v "^\($uuid\|$flags\|$seed\|$csum1\|$csum2\)" > "dump" &&
```

Comment:
Right, this depends on whether `char` is signed or unsigned, so this depends on the architecture. But other flags are architecture independent, so don't remove it completely. Maybe add something like this:
```
| sed 's/\(un\)\(signed_directory_hash\)/\2/'
```
And please adjust the commit message accordingly.

Reply:
Thanks for the review! I pushed a revised patch.